### PR TITLE
Issue rename app project

### PR DESCRIPTION
**Pull request by teleyinex**
Rename app to project as in the rest of the PyBossa ecosystems.

### CODE REVIEW

File: pbclient/__init__.py [hunk 1]
```diff
@@ -447,59 +480,60 @@ def delete_task(task_id):
...
 # Task Runs
 
-def get_taskruns(app_id, limit=100, offset=0):
-    """Returns a list of task runs for a given application ID
+def get_taskruns(project_id, limit=100, offset=0):
+    """Return a list of task runs for a given project ID.
 
-    :param app_id: PyBossa Application ID
-    :type app_id: integer
+    :param project_id: PyBossa Application ID
```
Comment (alejandrodob) [inline pbclient/__init__.py]: Project ID

File: pbclient/__init__.py [hunk 2]
```diff
@@ -36,12 +38,15 @@ def _pybossa_req(method, domain, id=None, payload=None, params=None):
...
+        r = requests.put(url, params=params, headers=headers,
+                         data=json.dumps(payload))
     elif method == 'delete':
-        r = requests.delete(url, params=params, headers=headers, data=json.dumps(payload))
-    #print r.status_code, r.status_code / 100
+        r = requests.delete(url, params=params, headers=headers,
+                            data=json.dumps(payload))
+    # print r.status_code, r.status_code / 100
     if r.status_code / 100 == 2:
         if r.text and r.text != '""':
```
Comment (alejandrodob) [inline pbclient/__init__.py]: this if can be deleted with no consequences, right?

Comment (teleyinex) [inline pbclient/__init__.py]: What do you mean?

Comment (alejandrodob) [inline pbclient/__init__.py]: We have one condition that, if holds, then we are returning exactly the same that if it is not (if r.text and r.text != "")

**Comment by alejandrodob** [comment]
:cake:

**Pull request merged**